config/logger.config

```diff
@@ -0,0 +1,28 @@
+---
+version: 1
+
+disable_existing_loggers: False
+
+formatters:
+    simple:
+        format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+handlers:
+    console:
+        class: logging.StreamHandler
+        level: DEBUG
+        formatter: simple
+        stream: ext://sys.stdout
+
+    file_handler:
+        class: logging.handlers.RotatingFileHandler
+        level: WARNING  # By default set to WARNING, but can be changed on runtime
+        formatter: simple
+        filename: messages.log
```

Comment:
Why call it `messages.log` instead of `product.log`? Isn't that a bit vague what it is referring to?